src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java

```diff
@@ -85,21 +90,21 @@ private void answerFile(TelegramRequest request) {
 						.caption(FILE_READY.getText())
 						.parseMode(MarkdownV2);
 
-				executeOrFail(answerWithFile);
+				execute(answerWithFile);
 			}
 		} catch (TelegramApiException e) {
-			LOGGER.warn("Unable to reply to {} with processed file", request.getDescription(), e);
+			LOGGER.atWarn().setCause(e).log("Unable to reply to {} with processed file", request.getDescription());
```

Comment:
I like this fluent API! I think we should be consistent in how we log, do you mind converting every log to this syntax?